openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -708,7 +541,6 @@ $().ready(function(){
         } else {
             dropdown.slideToggle(25);
             $(this).toggleClass('hover');
-            var offUser = $(this);
             $(document).mouseup(function(offUser){
```

Comment:
This reference is likely broken. mouseup is passed an "ev" and can't be used this way.

This  fix is correct but has likely uncovered a bug.

Reply:
@jdlrobson @mekarpeles is this safe to merge at this stage? I'm unsure of status / next steps for this one `offUser`?

Reply:
This can safely be removed.

The line below should probably be rewritten like so:
```
$(document).mouseup(function(ev){
if($(ev.target).parent("a").length==0){
```

but that can be done in a follow up.